client/src/actions/newGrant.ts

```diff
@@ -92,7 +92,9 @@ export const publishGrant =
     }
 
     application.credentials = formCredentials;
-    application.createdAt = Date.now();
+    if (!updating) {
```

Comment:
This could be changed to:

`if (!grantId) {`

Reply:
Thanks, it's more robust now